README.md

```diff
@@ -0,0 +1,85 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+iOS Playbook 📚
+==================================
+
+At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview of how we work:
+
+1. Who's in the team.
+2. Which squad each individual belongs to, and its availability.
+3. OSS maintained projects.
```

Comment:
OSS-maintained  (note the hyphen)

Reply:
As per your request for the rule, @RuiAAPeres, check the "noun+participate" case in https://en.oxforddictionaries.com/punctuation/hyphen.